### ISSUE TITLE
Extremely low FPS on Windows 

### ISSUE THREAD

**Issue by federico-digital** [hacksider/Deep-Live-Cam#805]
The LIVE runs at 1 FPS, I installed everything manually with no issues on Windows 10, I installed Visual Studio and CUDA also.

What am I doing wrong? My PC:

Microsoft Windows 10 Pro
Version	10.0.19045 Build 19045
System Type	x64-based PC
Processor	AMD Ryzen 7 3800X 8-Core Processor, 3893 Mhz, 8 Core(s), 16 Logical Processor(s)
Installed Physical Memory (RAM)	32.0 GB
Total Physical Memory	31.9 GB
Available Physical Memory	9.84 GB
Total Virtual Memory	63.9 GB
Available Virtual Memory	30.8 GB
Graphic Card: Radeon RX 560 Series

**Comment by hacksider** [comment]
CUDA won't work on radeon GPU

**Comment by hacksider** [comment]
Yes, it is normal to be slow but that's extremely slow on your side. My laptop can handle up to 4 fps using just cpu.

**Comment by irowbin** [comment]
I conducted some tests with and without the face enhancer feature. When I did not use the face enhancer, the frame rate was around 11-12 FPS. However, when I turned on the face enhancer, the frame rate dropped to just 2-3 FPS, which seemed incredibly slow. I checked the GPU VRAM usage, and it was not fully utilized, with the clock speed only reaching about 60-70%.

Could there be some tweaks to enhance this process?

Thank You!
